src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -243,3 +249,59 @@ def test_wait_for_event_with_timeout():
         with session.initiate():
             session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE, 0.5)
 
+
+def test_commit(single_channel_session):
+    non_default_current_limit = 0.00021
+    single_channel_session.current_limit = non_default_current_limit
+    single_channel_session.commit()
+    assert single_channel_session.current_limit == non_default_current_limit
+
+
+# TODO (bhaswath): Enable test after Pull request #467 is merged, which will enable the export signal enum
+'''
+def test_export_signal(single_channel_session):
+    expected_trigger_terminal = "//Engine0/MeasureTrigger"
+    single_channel_session.export_signal(nidcpower.Event.SOURCE_COMPLETE, expected_trigger_terminal)
+    assert expected_trigger_terminal == single_channel_session.source_complete_event_output_terminal
+'''
+
+
+def test_configure_digital_edge_measure_trigger_error(single_channel_session):
+    try:
+        single_channel_session.configure_digital_edge_measure_trigger("invalid string")
```

Comment:
Try not to write the tests so that they rely on error.
Don't have dead code in the tests. For example line 273 never executes.
You don't need to call initiate. Reading an attribute is enough to trigger a verify of the session. Calling initiate tries to verify, commit, and start the session.